### ISSUE TITLE
After running v11 build 117, ...

### ISSUE THREAD

**Issue by door43app** [unfoldingWord-dev/ts-desktop#1093]
Notes
======
After running v11 build 117, I loaded and ran v10. After again loading v11 the app hangs when choosing a source text. This was on a Mac.

Environment
======
Environment Key | Value
:--: | :--:
Version |11.1.0
Operating System | Darwin
Platform | darwin
Release | 15.6.0
Architecture | x64

Log History
======
```javascript
2017-02-24 15:32:35 I/ts-dashboard.html:1125:4: Feedback Sent
	https://api.github.com/repos/unfoldingWord-dev/ts-desktop/issues/1088
2017-02-24 15:35:23 I/ts-dashboard.html:1278:3: 
	Git Version 2.10.1 (Apple Git-78)
2017-02-24 15:35:23 W/migrator.js:24:2: Unable to migrate project .DS_Store
	Error: ENOTDIR: not a directory, open '/Users/cckozie/Library/Application Support/translationstudio/targetTranslations/.DS_Store/manifest.json'
2017-02-24 15:35:24 I/migrator.js:31:2: Migrated projects
	[
	  "ar_mat_text_ulb",
	  "en-x-demo1_1ch_text_ulb",
	  "en-x-demo1_1th_text_ulb",
	  "en-x-demo1_bible_tw",
	  "en-x-demo1_col_text_ulb",
	  "en-x-demo1_exo_text_ulb",
	  "en-x-demo1_phm_text_ulb",
	  "en-x-demo1_psa_text_ulb",
	  "en-x-demo2_php_text_reg",
	  "es-419_1th_text_ulb",
	  "es-419_2th_text_ulb",
	  "es-419_bible_tw",
	  "es-419_luk_text_ulb",
	  "qaa-x-1560c6_tit_text_reg",
	  "qaa-x-731b78_phm_text_reg"
	]
2017-02-24 15:35:25 E/projects.js:262:26: 
	TypeError: Cannot read property 'status' of null
2017-02-27 13:30:7 I/ts-dashboard.html:1278:3: 
	Git Version 2.10.1 (Apple Git-78)
2017-02-27 13:30:7 W/migrator.js:24:2: Unable to migrate project .DS_Store
	Error: ENOTDIR: not a directory, open '/Users/cckozie/Library/Application Support/translationstudio/targetTranslations/.DS_Store/manifest.json'
2017-02-27 13:30:8 I/migrator.js:31:2: Migrated projects
	[
	  "ar_mat_text_ulb",
	  "en-x-demo1_1ch_text_ulb",
	  "en-x-demo1_1th_text_ulb",
	  "en-x-demo1_bible_tw",
	  "en-x-demo1_col_text_ulb",
	  "en-x-demo1_exo_text_ulb",
	  "en-x-demo1_phm_text_ulb",
	  "en-x-demo1_psa_text_ulb",
	  "en-x-demo2_php_text_reg",
	  "es-419_1th_text_ulb",
	  "es-419_2th_text_ulb",
	  "es-419_bible_tw",
	  "es-419_luk_text_ulb",
	  "qaa-x-1560c6_tit_text_reg",
	  "qaa-x-731b78_phm_text_reg"
	]
2017-02-27 13:30:9 E/projects.js:262:26: 
	TypeError: Cannot read property 'status' of null
2017-02-27 17:11:0 I/ts-dashboard.html:1278:3: 
	Git Version 2.10.1 (Apple Git-78)
2017-02-27 17:11:0 W/migrator.js:24:2: Unable to migrate project .DS_Store
	Error: ENOTDIR: not a directory, open '/Users/cckozie/Library/Application Support/translationstudio/targetTranslations/.DS_Store/manifest.json'
2017-02-27 17:11:1 I/migrator.js:31:2: Migrated projects
	[
	  "ar_mat_text_ulb",
	  "en-x-demo1_1ch_text_ulb",
	  "en-x-demo1_1th_text_ulb",
	  "en-x-demo1_bible_tw",
	  "en-x-demo1_col_text_ulb",
	  "en-x-demo1_exo_text_ulb",
	  "en-x-demo1_phm_text_ulb",
	  "en-x-demo1_psa_text_ulb",
	  "en-x-demo2_php_text_reg",
	  "es-419_1th_text_ulb",
	  "es-419_2th_text_ulb",
	  "es-419_bible_tw",
	  "es-419_luk_text_ulb",
	  "qaa-x-1560c6_tit_text_reg",
	  "qaa-x-731b78_phm_text_reg"
	]
2017-02-27 17:11:3 E/projects.js:262:26: 
	TypeError: Cannot read property 'status' of null
2017-02-28 13:32:17 I/ts-dashboard.html:1278:3: 
	Git Version 2.10.1 (Apple Git-78)
2017-02-28 13:32:17 W/migrator.js:24:2: Unable to migrate project .DS_Store
	Error: ENOTDIR: not a directory, open '/Users/cckozie/Library/Application Support/translationstudio/targetTranslations/.DS_Store/manifest.json'
2017-02-28 13:32:18 I/migrator.js:31:2: Migrated projects
	[
	  "ar_mat_text_ulb",
	  "en-x-demo1_1ch_text_ulb",
	  "en-x-demo1_1th_text_ulb",
	  "en-x-demo1_bible_tw",
	  "en-x-demo1_col_text_ulb",
	  "en-x-demo1_exo_text_ulb",
	  "en-x-demo1_phm_text_ulb",
	  "en-x-demo1_psa_text_ulb",
	  "en-x-demo2_php_text_reg",
	  "es-419_1th_text_ulb",
	  "es-419_2th_text_ulb",
	  "es-419_bible_tw",
	  "es-419_luk_text_ulb",
	  "qaa-x-1560c6_tit_text_reg",
	  "qaa-x-731b78_phm_text_reg"
	]
2017-02-28 13:32:19 E/projects.js:262:26: 
	TypeError: Cannot read property 'status' of null
2017-02-28 18:5:8 I/ts-dashboard.html:1278:3: 
	Git Version 2.10.1 (Apple Git-78)
2017-02-28 18:5:8 W/migrator.js:24:2: Unable to migrate project .DS_Store
	Error: ENOTDIR: not a directory, open '/Users/cckozie/Library/Application Support/translationstudio/targetTranslations/.DS_Store/manifest.json'
2017-02-28 18:5:9 I/migrator.js:31:2: Migrated projects
	[
	  "ar_mat_text_ulb",
	  "en-x-demo1_1ch_text_ulb",
	  "en-x-demo1_1th_text_ulb",
	  "en-x-demo1_bible_tw",
	  "en-x-demo1_col_text_ulb",
	  "en-x-demo1_exo_text_ulb",
	  "en-x-demo1_phm_text_ulb",
	  "en-x-demo1_psa_text_ulb",
	  "en-x-demo2_php_text_reg",
	  "es-419_1th_text_ulb",
	  "es-419_2th_text_ulb",
	  "es-419_bible_tw",
	  "es-419_luk_text_ulb",
	  "qaa-x-1560c6_tit_text_reg",
	  "qaa-x-731b78_phm_text_reg"
	]
2017-02-28 18:5:11 E/projects.js:262:26: 
	TypeError: Cannot read property 'status' of null
2017-02-28 18:11:38 I/ts-dashboard.html:1202:3: 
	Git Version 2.10.1 (Apple Git-78)
2017-02-28 18:11:38 W/migrator.js:24:2: Unable to migrate project .DS_Store
	Error: ENOTDIR: not a directory, open '/Users/cckozie/Library/Application Support/translationstudio/targetTranslations/.DS_Store/manifest.json'
2017-02-28 18:11:39 I/migrator.js:31:2: Migrated projects
	[
	  "ar_mat_text_ulb",
	  "en-x-demo1_1ch_text_ulb",
	  "en-x-demo1_1th_text_ulb",
	  "en-x-demo1_bible_tw",
	  "en-x-demo1_col_text_ulb",
	  "en-x-demo1_exo_text_ulb",
	  "en-x-demo1_phm_text_ulb",
	  "en-x-demo1_psa_text_ulb",
	  "en-x-demo2_php_text_reg",
	  "es-419_1th_text_ulb",
	  "es-419_2th_text_ulb",
	  "es-419_bible_tw",
	  "es-419_luk_text_ulb",
	  "qaa-x-1560c6_tit_text_reg",
	  "qaa-x-731b78_phm_text_reg"
	]
2017-02-28 18:11:41 E/projects.js:260:26: 
	TypeError: Cannot read property 'status' of null
2017-02-28 18:24:46 I/ts-dashboard.html:1278:3: 
	Git Version 2.10.1 (Apple Git-78)
2017-02-28 18:24:46 W/migrator.js:24:2: Unable to migrate project .DS_Store
	Error: ENOTDIR: not a directory, open '/Users/cckozie/Library/Application Support/translationstudio/targetTranslations/.DS_Store/manifest.json'
2017-02-28 18:24:47 I/migrator.js:31:2: Migrated projects
	[
	  "ar_mat_text_ulb",
	  "en-x-demo1_1ch_text_ulb",
	  "en-x-demo1_1th_text_ulb",
	  "en-x-demo1_bible_tw",
	  "en-x-demo1_col_text_ulb",
	  "en-x-demo1_exo_text_ulb",
	  "en-x-demo1_phm_text_ulb",
	  "en-x-demo1_psa_text_ulb",
	  "en-x-demo2_php_text_reg",
	  "es-419_1th_text_ulb",
	  "es-419_2th_text_ulb",
	  "es-419_bible_tw",
	  "es-419_luk_text_ulb",
	  "qaa-x-1560c6_tit_text_reg",
	  "qaa-x-731b78_phm_text_reg"
	]
2017-03-01 10:12:20 I/ts-dashboard.html:1275:3: 
	Git Version 2.10.1 (Apple Git-78)
2017-03-01 10:12:21 W/migrator.js:24:2: Unable to migrate project .DS_Store
	Error: ENOTDIR: not a directory, open '/Users/cckozie/Library/Application Support/translationstudio/targetTranslations/.DS_Store/manifest.json'
2017-03-01 10:12:22 I/migrator.js:31:2: Migrated projects
	[
	  "ar_mat_text_ulb",
	  "en-x-demo1_1ch_text_ulb",
	  "en-x-demo1_1th_text_ulb",
	  "en-x-demo1_bible_tw",
	  "en-x-demo1_col_text_ulb",
	  "en-x-demo1_exo_text_ulb",
	  "en-x-demo1_phm_text_ulb",
	  "en-x-demo1_psa_text_ulb",
	  "en-x-demo2_php_text_reg",
	  "es-419_1th_text_ulb",
	  "es-419_2th_text_ulb",
	  "es-419_bible_tw",
	  "es-419_luk_text_ulb",
	  "qaa-x-1560c6_tit_text_reg",
	  "qaa-x-731b78_phm_text_reg"
	]
2017-03-01 12:7:49 E/ts-dashboard.html:872:3: Import Failed
	Error: Invalid resource container. Missing package.json
2017-03-01 12:33:58 I/ts-dashboard.html:1202:3: 
	Git Version 2.10.1 (Apple Git-78)
2017-03-01 12:33:59 W/migrator.js:24:2: Unable to migrate project .DS_Store
	Error: ENOTDIR: not a directory, open '/Users/cckozie/Library/Application Support/translationstudio/targetTranslations/.DS_Store/manifest.json'
2017-03-01 12:34:0 I/migrator.js:31:2: Migrated projects
	[
	  "ar_mat_text_ulb",
	  "en-x-demo1_1ch_text_ulb",
	  "en-x-demo1_1th_text_ulb",
	  "en-x-demo1_bible_tw",
	  "en-x-demo1_col_text_ulb",
	  "en-x-demo1_exo_text_ulb",
	  "en-x-demo1_phm_text_ulb",
	  "en-x-demo1_psa_text_ulb",
	  "en-x-demo2_php_text_reg",
	  "es-419_1th_text_ulb",
	  "es-419_2th_text_ulb",
	  "es-419_bible_tw",
	  "es-419_luk_text_ulb",
	  "qaa-x-1560c6_tit_text_reg",
	  "qaa-x-731b78_phm_text_reg"
	]
2017-03-01 12:34:1 E/projects.js:260:26: 
	TypeError: Cannot read property 'status' of null
2017-03-01 12:34:51 I/ts-dashboard.html:837:3: 
	Git Version 2.10.1 (Apple Git-78)
2017-03-01 12:34:51 W/migrator.js:24:2: Unable to migrate project .DS_Store
	Error: ENOTDIR: not a directory, open '/Users/cckozie/Library/Application Support/translationstudio/targetTranslations/.DS_Store/manifest.json'
2017-03-01 12:34:51 W/migrator.js:24:2: Unable to migrate project ar_mat_text_ulb
	Error: Failed to migrate project
2017-03-01 12:34:51 W/migrator.js:24:2: Unable to migrate project en-x-demo1_1ch_text_ulb
	Error: Failed to migrate project
2017-03-01 12:34:51 W/migrator.js:24:2: Unable to migrate project en-x-demo1_1th_text_ulb
	Error: Failed to migrate project
2017-03-01 12:34:51 W/migrator.js:24:2: Unable to migrate project en-x-demo1_bible_tw
	Error: Failed to migrate project
2017-03-01 12:34:51 W/migrator.js:24:2: Unable to migrate project en-x-demo1_col_text_ulb
	Error: Failed to migrate project
2017-03-01 12:34:51 W/migrator.js:24:2: Unable to migrate project en-x-demo1_exo_text_ulb
	Error: Failed to migrate project
2017-03-01 12:34:51 W/migrator.js:24:2: Unable to migrate project en-x-demo1_phm_text_ulb
	Error: Failed to migrate project
2017-03-01 12:34:51 W/migrator.js:24:2: Unable to migrate project en-x-demo1_psa_text_ulb
	Error: Failed to migrate project
2017-03-01 12:34:51 W/migrator.js:24:2: Unable to migrate project en-x-demo2_php_text_reg
	Error: Failed to migrate project
2017-03-01 12:34:51 W/migrator.js:24:2: Unable to migrate project es-419_1th_text_ulb
	Error: Failed to migrate project
2017-03-01 12:34:51 W/migrator.js:24:2: Unable to migrate project es-419_2th_text_ulb
	Error: Failed to migrate project
2017-03-01 12:34:51 W/migrator.js:24:2: Unable to migrate project es-419_bible_tw
	Error: Failed to migrate project
2017-03-01 12:34:51 W/migrator.js:24:2: Unable to migrate project es-419_luk_text_ulb
	Error: Failed to migrate project
2017-03-01 12:34:51 W/migrator.js:24:2: Unable to migrate project qaa-x-1560c6_tit_text_reg
	Error: Failed to migrate project
2017-03-01 12:34:51 W/migrator.js:24:2: Unable to migrate project qaa-x-731b78_phm_text_reg
	Error: Failed to migrate project
2017-03-01 12:34:51 I/migrator.js:31:2: Migrated projects
	[]
2017-03-01 12:34:52 E/projects.js:265:26: 
	TypeError: Cannot read property 'project_id' of undefined
2017-03-01 12:36:16 I/ts-dashboard.html:1275:3: 
	Git Version 2.10.1 (Apple Git-78)
2017-03-01 12:36:16 W/migrator.js:24:2: Unable to migrate project .DS_Store
	Error: ENOTDIR: not a directory, open '/Users/cckozie/Library/Application Support/translationstudio/targetTranslations/.DS_Store/manifest.json'
2017-03-01 12:36:17 I/migrator.js:31:2: Migrated projects
	[
	  "ar_mat_text_ulb",
	  "en-x-demo1_1ch_text_ulb",
	  "en-x-demo1_1th_text_ulb",
	  "en-x-demo1_bible_tw",
	  "en-x-demo1_col_text_ulb",
	  "en-x-demo1_exo_text_ulb",
	  "en-x-demo1_phm_text_ulb",
	  "en-x-demo1_psa_text_ulb",
	  "en-x-demo2_php_text_reg",
	  "es-419_1th_text_ulb",
	  "es-419_2th_text_ulb",
	  "es-419_bible_tw",
	  "es-419_luk_text_ulb",
	  "qaa-x-1560c6_tit_text_reg",
	  "qaa-x-731b78_phm_text_reg"
	]
2017-03-01 12:36:17 E/projects.js:268:26: 
	Error: no such table: legacy_resource_info
2017-03-01 12:36:17 E/projects.js:268:26: 
	Error: no such table: legacy_resource_info
2017-03-01 12:36:17 E/projects.js:268:26: 
	Error: no such table: legacy_resource_info
2017-03-01 12:36:18 E/projects.js:268:26: 
	Error: no such table: legacy_resource_info
2017-03-01 12:36:18 E/projects.js:268:26: 
	Error: no such table: legacy_resource_info
2017-03-01 12:36:18 E/projects.js:268:26: 
	Error: no such table: legacy_resource_info
2017-03-01 12:36:18 E/projects.js:268:26: 
	Error: no such table: legacy_resource_info
2017-03-01 12:36:18 E/projects.js:268:26: 
	Error: no such table: legacy_resource_info
2017-03-01 12:36:18 E/projects.js:268:26: 
	Error: no such table: legacy_resource_info
2017-03-01 12:36:18 E/projects.js:268:26: 
	Error: no such table: legacy_resource_info
2017-03-01 12:36:18 E/projects.js:268:26: 
	Error: no such column: p.source_language_id
2017-03-01 12:36:18 E/projects.js:268:26: 
	Error: no such table: legacy_resource_info
2017-03-01 12:36:18 E/ts-dashboard.html:1302:5: Error on startup
	Error: no such table: legacy_resource_info
	    at Error (native)
	    at ud.handleError (/Users/cckozie/Downloads/translationStudio-117.app/Contents/Resources/app/node_modules/door43-client/node_modules/sql.js/js/sql.js:464:445)
	    at ud.prepare (/Users/cckozie/Downloads/translationStudio-117.app/Contents/Resources/app/node_modules/door43-client/node_modules/sql.js/js/sql.js:463:235)
	    at query (/Users/cckozie/Downloads/translationStudio-117.app/Contents/Resources/app/node_modules/door43-client/lib/sqlite-helper.js:60:24)
	    at Object.findTranslations (/Users/cckozie/Downloads/translationStudio-117.app/Contents/Resources/app/node_modules/door43-client/lib/library.js:1062:27)
	    at Object.getTranslations (/Users/cckozie/Downloads/translationStudio-117.app/Contents/Resources/app/src/js/database.js:69:33)
	    at ts-dashboard.updateIndex (file:///Users/cckozie/Downloads/translationStudio-117.app/Contents/Resources/app/src/elements/ts-main/ts-dashboard.html:1218:50)
	    at file:///Users/cckozie/Downloads/translationStudio-117.app/Contents/Resources/app/src/elements/ts-main/ts-dashboard.html:1299:33
2017-03-01 12:36:18 E/ts-dashboard.html:1314:2: Error on startup
	Error: no such table: legacy_resource_info
2017-03-01 12:39:15 I/ts-dashboard.html:1275:3: 
	Git Version 2.10.1 (Apple Git-78)
2017-03-01 12:39:15 W/migrator.js:24:2: Unable to migrate project .DS_Store
	Error: ENOTDIR: not a directory, open '/Users/cckozie/Library/Application Support/translationstudio/targetTranslations/.DS_Store/manifest.json'
2017-03-01 12:39:16 I/migrator.js:31:2: Migrated projects
	[
	  "ar_mat_text_ulb",
	  "en-x-demo1_1ch_text_ulb",
	  "en-x-demo1_1th_text_ulb",
	  "en-x-demo1_bible_tw",
	  "en-x-demo1_col_text_ulb",
	  "en-x-demo1_exo_text_ulb",
	  "en-x-demo1_phm_text_ulb",
	  "en-x-demo1_psa_text_ulb",
	  "en-x-demo2_php_text_reg",
	  "es-419_1th_text_ulb",
	  "es-419_2th_text_ulb",
	  "es-419_bible_tw",
	  "es-419_luk_text_ulb",
	  "qaa-x-1560c6_tit_text_reg",
	  "qaa-x-731b78_phm_text_reg"
	]
2017-03-01 12:39:16 E/projects.js:268:26: 
	Error: no such table: legacy_resource_info
2017-03-01 12:39:16 E/projects.js:268:26: 
	Error: no such table: legacy_resource_info
2017-03-01 12:39:16 E/projects.js:268:26: 
	Error: no such table: legacy_resource_info
2017-03-01 12:39:17 E/projects.js:268:26: 
	Error: no such table: legacy_resource_info
2017-03-01 12:39:17 E/projects.js:268:26: 
	Error: no such table: legacy_resource_info
2017-03-01 12:39:17 E/projects.js:268:26: 
	Error: no such table: legacy_resource_info
2017-03-01 12:39:17 E/projects.js:268:26: 
	Error: no such table: legacy_resource_info
2017-03-01 12:39:17 E/projects.js:268:26: 
	Error: no such table: legacy_resource_info
2017-03-01 12:39:17 E/projects.js:268:26: 
	Error: no such table: legacy_resource_info
2017-03-01 12:39:17 E/projects.js:268:26: 
	Error: no such table: legacy_resource_info
2017-03-01 12:39:17 E/projects.js:268:26: 
	Error: no such column: p.source_language_id
2017-03-01 12:39:17 E/projects.js:268:26: 
	Error: no such table: legacy_resource_info
2017-03-01 12:39:17 E/ts-dashboard.html:1302:5: Error on startup
	Error: no such table: legacy_resource_info
	    at Error (native)
	    at ud.handleError (/Users/cckozie/Downloads/translationStudio-117.app/Contents/Resources/app/node_modules/door43-client/node_modules/sql.js/js/sql.js:464:445)
	    at ud.prepare (/Users/cckozie/Downloads/translationStudio-117.app/Contents/Resources/app/node_modules/door43-client/node_modules/sql.js/js/sql.js:463:235)
	    at query (/Users/cckozie/Downloads/translationStudio-117.app/Contents/Resources/app/node_modules/door43-client/lib/sqlite-helper.js:60:24)
	    at Object.findTranslations (/Users/cckozie/Downloads/translationStudio-117.app/Contents/Resources/app/node_modules/door43-client/lib/library.js:1062:27)
	    at Object.getTranslations (/Users/cckozie/Downloads/translationStudio-117.app/Contents/Resources/app/src/js/database.js:69:33)
	    at ts-dashboard.updateIndex (file:///Users/cckozie/Downloads/translationStudio-117.app/Contents/Resources/app/src/elements/ts-main/ts-dashboard.html:1218:50)
	    at file:///Users/cckozie/Downloads/translationStudio-117.app/Contents/Resources/app/src/elements/ts-main/ts-dashboard.html:1299:33
2017-03-01 12:39:17 E/ts-dashboard.html:1314:2: Error on startup
	Error: no such table: legacy_resource_info
2017-03-01 12:40:7 I/ts-dashboard.html:1275:3: 
	Git Version 2.10.1 (Apple Git-78)
2017-03-01 12:40:7 W/migrator.js:24:2: Unable to migrate project .DS_Store
	Error: ENOTDIR: not a directory, open '/Users/cckozie/Library/Application Support/translationstudio/targetTranslations/.DS_Store/manifest.json'
2017-03-01 12:40:8 I/migrator.js:31:2: Migrated projects
	[
	  "ar_mat_text_ulb",
	  "en-x-demo1_1ch_text_ulb",
	  "en-x-demo1_1th_text_ulb",
	  "en-x-demo1_bible_tw",
	  "en-x-demo1_col_text_ulb",
	  "en-x-demo1_exo_text_ulb",
	  "en-x-demo1_phm_text_ulb",
	  "en-x-demo1_psa_text_ulb",
	  "en-x-demo2_php_text_reg",
	  "es-419_1th_text_ulb",
	  "es-419_2th_text_ulb",
	  "es-419_bible_tw",
	  "es-419_luk_text_ulb",
	  "qaa-x-1560c6_tit_text_reg",
	  "qaa-x-731b78_phm_text_reg"
	]
2017-03-01 12:40:8 E/projects.js:268:26: 
	Error: no such table: legacy_resource_info
2017-03-01 12:40:8 E/projects.js:268:26: 
	Error: no such table: legacy_resource_info
2017-03-01 12:40:8 E/projects.js:268:26: 
	Error: no such table: legacy_resource_info
2017-03-01 12:40:9 E/projects.js:268:26: 
	Error: no such table: legacy_resource_info
2017-03-01 12:40:9 E/projects.js:268:26: 
	Error: no such table: legacy_resource_info
2017-03-01 12:40:9 E/projects.js:268:26: 
	Error: no such table: legacy_resource_info
2017-03-01 12:40:9 E/projects.js:268:26: 
	Error: no such table: legacy_resource_info
2017-03-01 12:40:9 E/projects.js:268:26: 
	Error: no such table: legacy_resource_info
2017-03-01 12:40:9 E/projects.js:268:26: 
	Error: no such table: legacy_resource_info
2017-03-01 12:40:9 E/projects.js:268:26: 
	Error: no such table: legacy_resource_info
2017-03-01 12:40:9 E/projects.js:268:26: 
	Error: no such column: p.source_language_id
2017-03-01 12:40:9 E/projects.js:268:26: 
	Error: no such table: legacy_resource_info
2017-03-01 12:40:9 E/ts-dashboard.html:1302:5: Error on startup
	Error: no such table: legacy_resource_info
	    at Error (native)
	    at ud.handleError (/Users/cckozie/Downloads/translationStudio-117.app/Contents/Resources/app/node_modules/door43-client/node_modules/sql.js/js/sql.js:464:445)
	    at ud.prepare (/Users/cckozie/Downloads/translationStudio-117.app/Contents/Resources/app/node_modules/door43-client/node_modules/sql.js/js/sql.js:463:235)
	    at query (/Users/cckozie/Downloads/translationStudio-117.app/Contents/Resources/app/node_modules/door43-client/lib/sqlite-helper.js:60:24)
	    at Object.findTranslations (/Users/cckozie/Downloads/translationStudio-117.app/Contents/Resources/app/node_modules/door43-client/lib/library.js:1062:27)
	    at Object.getTranslations (/Users/cckozie/Downloads/translationStudio-117.app/Contents/Resources/app/src/js/database.js:69:33)
	    at ts-dashboard.updateIndex (file:///Users/cckozie/Downloads/translationStudio-117.app/Contents/Resources/app/src/elements/ts-main/ts-dashboard.html:1218:50)
	    at file:///Users/cckozie/Downloads/translationStudio-117.app/Contents/Resources/app/src/elements/ts-main/ts-dashboard.html:1299:33
2017-03-01 12:40:9 E/ts-dashboard.html:1314:2: Error on startup
	Error: no such table: legacy_resource_info

```

**Comment by cckozie** [comment]
I clicked on an existing project that did not have any associated source text. When I clicked on the choose source icon, the app hangs with the Loading source options dialog displayed and animated.

**Comment by cdwhitfield73** [comment]
I believe that the real issue here happened way before you tried to select sources in a project.  Based on the error messages in this log, I found a couple issues with the app start up process in regards to moving from one version to another.  I fixed those issues which were the cause of the "Error: no such table: legacy_resource_info".  I think those errors caused other start up code not to complete successfully, which in turn caused the lock up when trying to select a source.  However, you will have to attempt this series of events again to be sure.

**Comment by cckozie** [comment]
See additional info in issue #1102

**Comment by cdwhitfield73** [comment]
After much more testing, I concluded that the fixes I made for this issue the first time were still not effective.  Although it was overwriting the new index file during startup, it was still not using the new index file in the app until the app was restarted again.  This was causing some projects to not show up in the list as well as the source modal to not load.  I have added more error handling to account for all these possibilities.  Projects will still show up in the list even if a source in that project fails.  If loading the source modal fails, it will give the user a message, suggesting they restart the app.  A few other possible error conditions were dealt with as well.  However, none of this should be needed because I also fixed the root cause and now it does actually use the new database when starting up during an upgrade to 11.1
